### PR TITLE
introduce a default.cfg based on the mainnet.cfg without snap sync

### LINKED ISSUE #4289: Don't make mainnet.cfg the default
**Describe the bug**
We've introduced an unexpected breaking change in 1.13.4:
In case you do not use your own configuration file, _mainnet.cfg_ settings are used by default and since the update, SnapSync is enabled by default for the mainnet configuration, which means it's enabled for your node too.
It will cause a problem in networks without SnapSync supporting nodes.

**Expected behavior**
Nodes without a config file shouldn't use the _mainnet.cfg_ file as a default.

### PR DESCRIPTION
**Pull request by smartprogrammer93**
Resolves #4289

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [x] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by LukaszRozmej** [comment]
Are we progressing with this?

**Comment by smartprogrammer93** [comment]
@dceleda i think we need your opinion here since you opened the issue

**Review by flcl42 — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/configs/default.cfg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "AdditionalRpcUrls": [
+      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
+    ]
+  },
+  "Merge": {
```
Comment (jmederosalvarado) [inline src/Nethermind/Nethermind.Runner/configs/default.cfg:37]: this is no longer needed

File: src/Nethermind/Nethermind.Runner/configs/default.cfg [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "Metrics": {
+    "NodeName": "Mainnet"
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "AdditionalRpcUrls": [
```
Comment (jmederosalvarado) [inline src/Nethermind/Nethermind.Runner/configs/default.cfg:33]: I think it would be better to use `EnginePort`

**Review by jmederosalvarado — COMMENTED** [review]
I don't understand the need for this config file, even if we decide that we want a config file that only changes SnapSync to false by default I don't agree with the naming

**Review by marcindsobczak — COMMENTED** [review]
I don't understand what is the purpose of this change. We will still have mainnet as default network, just in different config file and without SnapSync enabled. IMO many users will use this default config and will be frustrated by long fast-syncing time. And to use network other than mainnet it is still a need to specify it's config file. For network not supporting SnapSync we have it disabled in config files.

**Comment by dceleda** [comment]
https://github.com/NethermindEth/nethermind/issues/4281

**Pull request closed without merging**